spec/index.bs

```diff
@@ -9329,6 +9360,11 @@ regardless of the given {{GPUCanvasConfiguration}}.{{GPUCanvasConfiguration/devi
 initially set to: &laquo;{{GPUTextureFormat/"bgra8unorm"}}, {{GPUTextureFormat/"rgba8unorm"}},
 {{GPUTextureFormat/"rgba16float"}}&raquo;.
 
+Note: Canvas configuration cannot use `srgb` formats like {{GPUTextureFormat/"bgra8unorm-srgb"}}.
+Instead, use the non-`srgb` equivalent ({{GPUTextureFormat/"bgra8unorm"}}), specify the `srgb`
+format in the {{GPUCanvasConfiguration/viewFormats}}, and use {{GPUTexture/createView()}} to create
+a view with an `srgb` format.
```

Comment:
I'm considering reversing this and adding srgb formats back to the `supported context formats` so that on older D3D and Vulkan drivers we may be able to avoid enabling reinterpretation (`TYPELESS`/`VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT`) just for this common case.
https://github.com/gpuweb/gpuweb/discussions/2537#discussioncomment-2062936